src/components/Rockets.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Button, Image } from 'react-bootstrap';
+import { addReservationRocket, removeReservationRocket } from '../redux/api/api';
+import './style/Rockets.css';
+
+function Rockets() {
+  const { rocketsReducer } = useSelector((state) => state);
+  const { rockets } = rocketsReducer;
+  const [rocketsDisplay, setRocketsDisplay] = useState(null);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (rockets !== rocketsDisplay) {
+      setRocketsDisplay(rockets);
+    }
+  });
+
+  function checkButton(status) {
+    if (status.reserved === false) {
+      return (
+        <Button variant="primary" onClick={() => dispatch(addReservationRocket(status))}>Reserve Rocket</Button>
+      );
+    }
+    if (status.reserved === true) {
+      return (
+        <Button variant="outline-secondary" onClick={() => dispatch(removeReservationRocket(status))}>Cancel Reservation</Button>
+      );
+    }
+    return (
+      <h3>Member?</h3>
+    );
+  }
+
+  function checkStatus(status) {
+    if (status.reserved === true) {
+      return (
+        <p className="specialFloat">Reserved</p>
+      );
+    }
+    if (status.reserved === false) {
+      return (
+        <p className="special" />
+      );
+    }
+    return (
+      <h3>Member?</h3>
+    );
+  }
+
+  if (rocketsDisplay) {
+    return (
+      <div className="border-top border-2">
+        {rocketsDisplay.map((element) => (
+          <div key={element.id} className="rocketContainer">
+            <Image src={element.flickr_images[0]} alt="rocket" className="w-25 ms-2 me-3 imageRocket" />
+            <div className="textRocket">
+              <h1>{element.name}</h1>
+              <div className="specialRocket">
+                {checkStatus(element)}
+                <p className="desc">{element.description}</p>
```

Comment:
- [ ] Would you please double-check on the styling of this paragraph on the 'Rockets' page, because it's slightly different from the original design, there's a non-pleasant gap to the eyes that'd be much nicer if it's gone 😄 🙏 
![Screenshot](https://user-images.githubusercontent.com/47625407/132037962-08acfae5-f21e-4c57-aae1-389a2012566f.png)

_You can check the image right [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=The%20Rockets%20section%20displays%20a%20list%20of%20all%20available%20SpaceX%20rockets.%20Users%20can%20book%20each%20rocket%20by%20clicking%20the%20reservation%20button%20or%20cancel%20the%20previously%20made%20booking.%20The%20same%20layout%20is%20used%20to%20form%20the%20Dragons%20section%20%5Badded%20only%20if%20your%20team%20has%203%20members%5D.)._